### PR TITLE
Spin off chart studio related content from r/getting-started into its own post

### PR DESCRIPTION
**Pull request by jdamiba**
The purpose of this PR is to remove the Chart Studio related content from the `r/getting-started` post. This content will be spun off into a post titled `getting-started-with-chart-studio` which will be in `_posts/r/chart-studio`.

Also, the wording of the doc has been changed to make it clearer that `plotly` is an R package.

### CODE REVIEW

**Comment by nicolaskruchten** [comment]
This looks great, but the spun-off content should live in the documentation repo with the rest of the chart-studio stuff please. Sorry I wasn't clearer about that!

File: r/2015-07-30-getting-started.Rmd [hunk 1]
```diff
@@ -16,130 +16,48 @@ knitr::opts_chunk$set(message = FALSE, warning=FALSE)
...
+### Note For `RStudio` Users
 
-<iframe style="border: none; width: 100%; height: 500px;" src="https://plot.ly/~chriddyp/1799.embed"></iframe>
+`RStudio` users should ensure that they are using the latest [RStudio release](https://www.rstudio.com/products/rstudio/download/) in order to ensure compatibility with the `htmlwidgets` R package.
 
-Plotly graphs are interactive. Click on legend entries to toggle traces, click-and-drag on the chart to zoom, double-click to autoscale, shift-and-drag to pan.
+## Rendering Charts
 
-#### Initialization for Online Plotting
+By default, the `plotly` R package runs locally in your web browser or in the ` viewer.
```
Comment (nicolaskruchten) [inline r/2015-07-30-getting-started.Rmd]: stray backtick here ?

Comment (jdamiba) [reply r/2015-07-30-getting-started.Rmd]: Supposed to say `RStudio` viewer. Fixed!

**Comment by nicolaskruchten** [comment]
Looks great, thanks for all the extra text and links! 💃

**Pull request merged**